src/libs/ui/src/atoms/ProductDetailsPageFavoriteButton/ProductDetailsPageFavoriteButton.styles.scss

```diff
@@ -0,0 +1,64 @@
+.favorite-icon {
+  height: 23px;
+  width: 24px;
+  background-size: cover;
+  background-position: center;
+}
+
+/* Default outline icon */
+.favorite-icon-outline {
+  background-image: url('https://www.pcaskinpro.com/products/_next/static/media/favorite-blue.4800f7c0.svg');
+}
+
+/* Filled icon when favorite is clicked */
+.favorite-icon-filled {
+  background-image: url('https://www.pcaskinpro.com/products/_next/static/media/favorite-blue-filled.ea8abfb4.svg');
+}

```

Comment:
please use SVG Atoms , if you need any help please ask to @Nishant8209, and all images